config/initializers/devise.rb

```diff
@@ -7,7 +7,6 @@
   require 'devise/orm/active_record'
   config.allow_unconfirmed_access_for = 0.days
   config.case_insensitive_keys = []
-  config.confirm_within = 24.hours
```

Comment:
I think this is related to account/email confirmation rather than account deletion